.eslintrc

```diff
@@ -9,6 +9,7 @@
   "plugins": ["tsdoc"],
   "rules": {
     "@typescript-eslint/member-ordering": "off",
+    "@typescript-eslint/return-await": "off",
```

Comment:
Why do we want this and how is it related?

Reply:
It looks like a leftover from one of my attempts 🤔 We can remove it for now

Reply:
Done